File: src/com/fusenetworks/fuse/listener/NoHunger.java
package com.fusenetworks.fuse.listener;
 
import static com.fusenetworks.fuse.Fuse.plugin;
import org.bukkit.event.EventHandler;
import org.bukkit.event.EventPriority;
import org.bukkit.event.Listener;
import org.bukkit.event.entity.FoodLevelChangeEvent;

public class NoHunger implements Listener
{
    String hungerenabled = plugin.getConfig().getString("server.hunger_enabled");
    @EventHandler(priority = EventPriority.MONITOR)
    public void onFoodLevelChange(FoodLevelChangeEvent event)
    {
        if (!hungerenabled.equalsIgnoreCase("true"))
        {
        event.setFoodLevel(20);
        }
    }
}

File: src/com/fusenetworks/fuse/listener/NoHunger.kt
package com.fusenetworks.fuse.listener

import org.bukkit.Bukkit
import org.bukkit.event.EventHandler
import org.bukkit.event.EventPriority
import org.bukkit.event.Listener
import org.bukkit.event.entity.FoodLevelChangeEvent

object NoHunger : Listener {
    internal var hungerenabled =  Bukkit.getPluginManager().getPlugin("Fuse").getConfig().getString("server.hunger_enabled")
    @EventHandler(priority = EventPriority.MONITOR)
    fun onFoodLevelChange(event: FoodLevelChangeEvent) {
        if (!hungerenabled.equals("true", ignoreCase = true)) {
            event.foodLevel = 20
        }
    }
}